lib/commentOnOpenedPullRequest.js

```diff
@@ -67,6 +67,7 @@ commentOnOpenedPullRequest._implementation = function (pullRequestFilesUrl, pull
 
             var askAboutChanges = commentOnOpenedPullRequest._askAboutChanges(files, baseBranch);
             var askAboutThirdParty = commentOnOpenedPullRequest._askAboutThirdParty(files, repositorySettings.thirdPartyFolders);
+            var askAboutTests = commentOnOpenedPullRequest._askAboutTests(files, repositorySettings);
```

Comment:
Why aren't we only passing `repositorySettings.unitTestPath`, like with `askAboutThirdParty`?